config/default/manager_patch.yaml

```diff
@@ -51,7 +51,7 @@ spec:
         - name: manager
           envFrom:
             - configMapRef:
-                name: system
```

Comment:
The name of the ConfigMap should likely still be "system" within the "substratus" namespace.

Reply:
fixed